### ISSUE TITLE
How to disable the Captcha on registration?

### ISSUE THREAD

**Issue by ruzakuku** [WWBN/AVideo#8255]
How to disable the Captcha on registration?

**Comment by DanielnetoDotCom** [comment]
no, it is impossible, if you do it you will open a change for bots to create fake users easily.

I am not willing to remove this feature.

**Comment by ruzakuku** [comment]
Remove of course no but put as an option?
